pkg/pipeline/ChartService.go

```diff
@@ -47,6 +48,39 @@ import (
 	"time"
 )
 
+type NameIncludesExcludes struct {
+	Names []string `json:"names"`
+}
+type Specs struct {
```

Comment:
why plural?

Reply:
the 'names' field is plural because it will have the includes/excludes array, specs is changed to spec